lib/cmd/compile/scripts.js

```diff
@@ -482,7 +484,9 @@ function compile(options = {}) {
              * would be removed from this deps file upon watch recompilation.
              * @see https://github.com/clay/claycli/issues/116#issuecomment-454110714
              */
-            buildScripts(bundleOptions.cache.files, bundleOptions)
+            // limiting entrypoints to kiln directory files
+            let cacheFiles = bundleOptions.cache.files.filter(item => item.match(/\/kiln\//));
+            buildScripts( cacheFiles, bundleOptions)
```

Comment:
something in array of browserify-cache files/entry-points was interfering with kiln-plugin compilation. filtering that set of entry-points to just `/kiln/` directory files seems to resolve the issue